### PR TITLE
MobilePolka: Unveiling Polkadot's Depths on the Go

### PR DESCRIPTION
**Pull request by EsubalewAmenu**
#### Project Abstract

Our project addresses the challenge of limited mobile access to comprehensive Polkadot blockchain data, hindering users' understanding and exploration of network activities. Our solution involves developing an intuitive Polkadot Explorer mobile app, empowering users to seamlessly analyze blockchain data. By enhancing accessibility and understanding, this app aims to significantly improve the user experience in navigating the Polkadot network.

#### Grant [level](https://github.com/w3f/Grants-Program#level_slider-levels)

- [x] **Level 1**:  Up to $10,000, 2 approvals
- [ ] **Level 2**:  Up to $30,000, 3 approvals
- [ ] **Level 3**:  Unlimited, 5 approvals (for >$100k: Web3 Foundation Council approval)

#### Application Checklist

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied and aptly renamed (`project_name.md`).
- [x] I have read the [application guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/grant_guidelines_per_category.md).
- [x] Payment details have been provided (bank details via email _or_ Polkadot (USDC & USDT) address in the application).
- [x] I am aware that, in order to receive a grant, I (and the entity I represent) have to successfully complete a KYC/KYB check.
- [x] The software delivered for this grant will be released under an open-source license specified in the application.
- [ ] The initial PR contains only one commit (squash and force-push if needed).
- [x] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery#process) (see the [announcement guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/announcement-guidelines.md)).
- [ ] I prefer the discussion of this application to take place in a private Element/Matrix channel. My username is: `@_______:matrix.org` (change the homeserver if you use a different one)

### CODE REVIEW

**Comment by EsubalewAmenu** [comment]
I have read and hereby sign the Contributor License Agreement.

**Review by semuelle — CHANGES_REQUESTED** [review]
Thanks for the application, @EsubalewAmenu. A couple of questions:

- [We ask for mockups and/or wireframes for any UI project](https://grants.web3.foundation/docs/Support%20Docs/grant_guidelines_per_category#ui-development). Can you provide anything in this regard? Especially since you included screenshots that are clearly not intended for mobile screens, it would be interesting to see how the data would be displayed on small screens.
- I understand that your are building an Android app, so I assume you are using Java. Regardless, could you add a tech stack? For example, what API are you using the get the data?
- I'm not familiar with any mobile explorer apps, either in the Polkadot ecosystem or other ecosystems. Have you done any research on why that might be?

File: applications/mobile_polka.md [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+- **Esubalew Amenu**: (Team leader) [BSc in Computer Science](https://drive.google.com/file/d/1jdVCjDgwIPjMnBgvw2jS_KkyEl_SBKYO/view?usp=drive_link), [MSc in Software Engineering](https://drive.google.com/file/d/1srLK67BwraL95o8DOaD0v0eRPVfsUfXc/view?usp=drive_link), BA in Management. Co-founder of DataScience and Herma computing plc. Over 8 years of experience in blockchain and fintech industries. Successfully led multiple projects from conception to implementation, ensuring timely delivery and client satisfaction.
+- **Eyasu Bihanu**: Has a strong background in blockchain applications.
+- **Minilik Eshetu**: Experienced backend developer proficient in building robust and scalable systems for blockchain applications.
+- **Khalid Abdurahiman**: Skilled mobile app developer adept at creating intuitive and feature-rich applications for various platforms, specializing in blockchain solutions.
+- **Tadesse Duressa**:Talented UI/UX designer with a focus on crafting engaging and intuitive user experiences for blockchain applications, ensuring seamless navigation and user satisfaction.
+
+### Contact
+
+- **Contact Name**: Esubalew Amenu
+- **Contact Email**: esubalew.a2009@gmail.com
```
Comment (semuelle) [inline applications/mobile_polka.md:149]:
```suggestion
- **Contact Name:** Esubalew Amenu
- **Contact Email:** esubalew.a2009@gmail.com
```

**Comment by EsubalewAmenu** [comment]
Thanks @semuelle for your inquiry! 

1. Regarding the UI aspect, while the screenshots provided were from [explorer.polkascan.io](https://explorer.polkascan.io/polkadot/block) to illustrate our proposal, we typically initiate our projects with Figma designs for UI/UX. We'll be commencing the design phase once funding is secured. For reference, you can view sample Figma designs from our past projects such as [Mindplex app](https://www.figma.com/file/1apZqXbb1QLlUskvZ0KLkN/Mindplex-app?type=design&node-id=0%3A1&mode=design&t=3rBpgERbTFZ7s8Px-1), [ExpertsWay app](https://www.figma.com/file/M7miP6ImsnbnTRJzMPD9a3/Experts-Way?node-id=0%3A1&t=UO1fVgY2whCg5EtK-1), [HealthPilot app](https://www.figma.com/file/pP5kmBTi9Ulx6p4B9irlJg/Untitled?type=design&node-id=0%3A1&t=83DqDfwwMEbUhvVH-1), and [Millions recipe app](https://www.figma.com/file/bH6JFFMqIS6QPnVznfpFPv/Untitled?type=design&node-id=8-253&mode=design&t=YLd4leDIDITfikZM-0).

2. As for the tech stack, we'll be using Flutter to ensure cross-platform compatibility for both Android and iOS. Initially, the app will be developed on Android Studio with plans to expand to iOS. Our data will be sourced from [Subsquid APIs](https://www.subsquid.io/), which are currently utilized by [explorer.polkascan.io](https://explorer.polkascan.io/polkadot/block).

3. Regarding mobile explorer apps, while there may not be options available specifically tailored to the Polkadot ecosystem, there are numerous examples across various blockchain networks. Notable examples for Ethereum network include [Ethereum Block Explorer](https://play.google.com/store/apps/details?id=nl.approvedapps.ethexplorer&hl=en&gl=US), [Blockchain explorer for Ethereum](https://bankex-scan.en.aptoide.com/app), [Ethereum Block Explorer ERC20](https://apps.apple.com/us/app/ethereum-block-explorer-erc20/id1261772749), and [Blockchair: Block Explorer](https://play.google.com/store/apps/details?id=com.bdc.blockchair.app&hl=en&gl=US). 

Our research indicates a mobile explorer app is essential for the Polkadot network, and we aim to address this need.

Thanks for your #support!

**Review by semuelle — CHANGES_REQUESTED** [review]
> 1. We'll be commencing the design phase once funding is secured.

To clarify, we require wireframes or mockups in the grant application.

**Comment by EsubalewAmenu** [comment]
Sure, my team is working on it day and night. It'll be ready soon. I'll share you the link once it's ready

**Comment by EsubalewAmenu** [comment]
Hello @semuelle, the [version 0.1](https://www.figma.com/file/ZtWv6MjWyRwnSK1AeDdAdo/MobilePolka?type=design&node-id=0-1&mode=design&t=JMvJoxSMtniCEgdc-0) of the design is ready! Please take a look by [clicking here](https://www.figma.com/file/ZtWv6MjWyRwnSK1AeDdAdo/MobilePolka?type=design&node-id=0-1&mode=design&t=JMvJoxSMtniCEgdc-0) and let us know if there is anything we need to add. 

Thank you.

**Review by semuelle — CHANGES_REQUESTED** [review]
Thanks for the update, @EsubalewAmenu. Could you add the figma link to the application document? I will then share it with the rest of the committee. To be honest, I'm not convinced that a native mobile explorer is needed, but other in the committee might disagree.

**Comment by semuelle** [comment]
Thanks for the updates, @EsubalewAmenu. As I said, I'll share the application with the rest of the committee.

**Review by takahser — CHANGES_REQUESTED** [review]
@EsubalewAmenu thanks for your interest in our grants program. I've got a couple of questions regarding your proposal:

- Could you share how you came up with that conclusion? For example, what methods did you use to conduct the research?
  > Our research indicates a mobile explorer app is essential for the Polkadot network, and we aim to address this need.
- Could you share which specific app(s) you're referring to here?
  > Our previous mobile app projects showcase our proficiency, with successful applications boasting high download numbers (100,000+) and active user engagement(50,000+).
- Could you explain where and how the data will be stored/cached, if that's the case? Or are you planning to fetch the data directly from the nodes?

**Comment by EsubalewAmenu** [comment]
- We analyzed existing [mobile applications](https://play.google.com/store/apps/details?id=nl.approvedapps.ethexplorer&hl=en&gl=US) on platforms like Google Play, noting significant download figures exceeding 10,000 for similar apps. Additionally, we conducted an oral survey within the [Ethiopian blockchain community](https://web.telegram.org/a/#-1001350725554_1), specifically gathering insights from the [CodeNight group](https://web.telegram.org/a/#-1001204209308), which revealed a strong preference for mobile explorer apps. Furthermore, our participation in the WEB3 Ethiopia Seminar on February 4, 2024, at the Magnolia Hotel provided additional validation of the need for such an app.

- Particularly the "[1-12 Textbooks](https://play.google.com/store/apps/details?id=com.herma.apps.textbooks)" app, which showcased our proficiency in developing applications with substantial download numbers

- As mentioned earlier (above), we plan to utilize [Subsquid APIs](https://www.subsquid.io/), ensuring real-time data access for the mobile explorer app. Additionally, our approach aligns with industry standards, utilizing the same data source as [explorer.polkascan.io](explorer.polkascan.io). This ensures reliable and up-to-date data retrieval for optimal user experience.

Thank you @takahser

**Comment by EsubalewAmenu** [comment]
photo1706173775(1)
photo1708241636(1)
photo1708241629(1)
photo1708241636

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks a lot for the application and all the effort you put into this. However, the grants committee decided today not to go ahead with it. Mainly because websites like https://www.subscan.io/ look perfectly fine on mobile, and we don't see the need for an additional mobile app. Independent of it, we wish you all the best for your project.

**Pull request closed without merging**